easyopac.make

```diff
@@ -606,6 +606,52 @@ projects[easyopac_redirect][download][url]    = "git@github.com:easySuite/easyop
 ;projects[easyopac_redirect][download][tag]    = ""
 projects[easyopac_redirect][download][branch] = "development"
 
+; Culture
+projects[culture][type]             = "theme"
+projects[culture][subdir]           = ""
+projects[culture][download][type]   = "git"
+projects[culture][download][url]    = "git@github.com:easySuite/culture.git"
+projects[culture][download][branch] = "develop"
+
+projects[culture_frontend][type]             = "module"
```

Comment:
I'm very unsure about the changes in this module, however its not related to this PR.

Reply:
I think this module should be used to keep culture-specific exports.